projects/mercury/src/metadata/views/MetadataViewTable.js

```diff
@@ -131,16 +154,23 @@ export const MetadataViewTable = (props: MetadataViewTableProperties) => {
         const displayValue = (value || []).map(v => ((column.type === 'Date') ? formatDate(v.value) : v.label)).join(', ');
 
         return (
-            <TableCell key={column.name}>
+            <TableCell key={column.name} onClick={onClickHandler}>
                 <span className={classes.cellContents}>{displayValue}</span>
             </TableCell>
         );
     };
 
+    const checkedCount = (Object.values(checkboxes) ? Object.values(checkboxes).reduce((sum, item) => (item === true ? sum + 1 : sum), 0) : 0);
+    const rowCount = data.rows && data.rows.length;
+
     return (
         <Table data-testid="results-table" size="small" stickyHeader={!loading}>
             <TableHead>
                 <TableRow>
+                    <TableCell style={{padding: 0}}>
+                        <Checkbox id="checkAll" key={Math.random()} onChange={handleCheckallChange} defaultChecked={rowCount > 0 && checkedCount === rowCount} />
+                    </TableCell>
+                    <TableCell />
```

Comment:
Why is this empty cell added in the header?
There is some issue with the table now. The headers are shifted by one column (header does not match the cells in the column, e.g. below subject header is above gender column):
![Screenshot from 2022-09-23 12-42-05](https://user-images.githubusercontent.com/20908736/191944647-39766fe8-20ec-43a1-90a7-ae710f0d5af8.png)

Also there s something wrong with rows styling - the gray line below every row is now only covering part of the row.